api/notd/token_metadata_processor.py

```diff
@@ -95,7 +95,9 @@ async def retrieve_token_metadata(self, registryAddress: str,tokenId: str) -> Re
                 basestr = tokenMetadataUri.replace('data:application/json;base64,', '', 1)
                 base64Metadata = basestr.encode('utf-8')
                 metadataBytes = base64.b64decode(base64Metadata)
-                tokenMetadataDict = json.loads(metadataBytes.decode('utf-8'))
+                metadataString = metadataBytes.decode('utf-8')
```

Comment:
if this is specific for this contract we should really wrap it in some conditionals or even just not include it. I'll mark this PR as WIP for now